### PR TITLE
DEV 2058 get version with setuptools_scm

### PR DESCRIPTION
**Pull request by qiaouchicago**

### CODE REVIEW

File: indexd/index/blueprint.py [hunk 1]
```diff
@@ -513,8 +513,8 @@ def version():
     """
 
     base = {
-        "version": VERSION,
-        "commit": COMMIT,
+        "version": get_version(),
+        "commit": "depreciated",
```
Comment (wrkhenddher) [inline indexd/index/blueprint.py:517]: typo?
deprecated.

**Review by wrkhenddher — APPROVED** [review]

File: indexd/index/blueprint.py (hunk 1 above)
Comment (mpsolano) [inline indexd/index/blueprint.py:517]: Should this be "deprecated"?

Comment (qiaouchicago) [reply indexd/index/blueprint.py]: typo

**Review by stilesj-uchicago — APPROVED** [review]

**Review by mpsolano — APPROVED** [review]
Looks good!

**Pull request merged**